src/amo/components/DownloadFirefoxButton/index.js

```diff
@@ -0,0 +1,57 @@
+/* @flow */
+import classNames from 'classnames';
+import React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import translate from 'core/i18n/translate';
+import type { UserAgentInfoType } from 'core/reducers/api';
+import Button from 'ui/components/Button';
+
+
+type PropTypes = {
+  className?: 'string',
+  i18n: Object,
+  userAgentInfo: UserAgentInfoType,
+}
+
+export const DownloadFirefoxButtonBase = ({
+  className,
+  i18n,
+  userAgentInfo,
+// eslint-disable-next-line indent
```

Comment:
It looks like this is fixed in `eslint` 4.4 https://github.com/eslint/eslint/pull/9027 Maybe we should just upgrade it :)

Reply:
Works for me, I'll try that 😄

Reply:
looks like @muffinresearch took care of it 🍰 https://github.com/mozilla/addons-frontend/pull/3046